src/routes/InventoryDetail.cy.js

```diff
@@ -51,11 +51,9 @@ describe('renders correctly', () => {
     cy.get('[data-cy="compliance-tab"]')
       .parent('.pf-c-tabs__item.pf-m-disabled')
       .should('have.length', 1);
-
     cy.get('[data-cy="vulnerabilities-tab"]')
       .parent('.pf-c-tabs__item')
-      .click();
-    cy.get('[data-cy="vulnerability-edge-prompt"]').should('have.length', 1);
+      .should('have.length', 1);
```

Comment:
What is the reason for removing the `click`ing here?

Reply:
before we had a link to go to edge management vulnerability tab, now we are opening the list of cves